presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoUpgrader.java

```diff
@@ -30,16 +33,19 @@
 {
     private static final Logger LOGGER = LogManager.getLogger(PrestoUpgrader.class);
     private static final int SUBPROCESS_TIMEOUT = 150;
+    private static final String CONFIG_DIR = "/etc/presto";
 
     private final PackageType packageType;
     private final URL urlToFetchPackage;
     private final boolean disableDependencyChecking;
+    private final boolean preserveConfigs;
 
-    public PrestoUpgrader(PackageType packageType, URL urlToFetchPackage, boolean disableDependencyChecking)
+    public PrestoUpgrader(PackageType packageType, URL urlToFetchPackage, boolean disableDependencyChecking, boolean preserveConfigs)
```

Comment:
This is starting to get well into the realm of the [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap).
But if you'd rather not change them, please add Javadoc comments documenting the any opaque boolean parameters.